src/main/java/io/games/simple/main/StartController.java

```diff
@@ -0,0 +1,19 @@
+package io.games.simple.main;
+
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+@Controller
+public class StartController {
+
+    @GetMapping("/login")
+    public String login(Model model){
+        return "login";
+    }
+    @GetMapping("/")
+    public String register(Model model){
+        return "register";
+    }
+}
```

Comment:
I think better option will be to have two separate controllers one for login and another for registration, because StartController name can be misleading and in my opinion better to have separate controllers for each view.